pocketbone-kicad.kicad_pcb

```diff
@@ -4608,6 +4609,9 @@
       (xy 4.817536 3.229429) (xy 4.839415 3.147786)) (layer B.SilkS) (width 0.01))
   )
 
+  (gr_text %%gitversion%% (at 123.825 112.395 90) (layer B.SilkS) (tstamp 58CDFF39)
```

Comment:
BTW, I love the idea of the silkscreen indicating the PCB revision. Was that one of the main thrusts?